02_configure_host.sh

```diff
@@ -37,6 +37,12 @@ export REPO_PATH=${WORKING_DIR}
 sync_repo_and_patch metal3-dev-env https://github.com/metal3-io/metal3-dev-env.git https://patch-diff.githubusercontent.com/raw/metal3-io/metal3-dev-env/pull/37.patch
 VM_SETUP_PATH="${REPO_PATH}/metal3-dev-env/vm-setup"
 
+ANSIBLE_FORCE_COLOR=true ansible-playbook \
+  -e "working_dir=$WORKING_DIR" \
+  -e "virthost=$HOSTNAME" \
+  -i ${VM_SETUP_PATH}/inventory.ini \
+  -b -vvv ${VM_SETUP_PATH}/install-package-playbook.yml
+
```

Comment:
I think to align with the metal3-dev-env PR this should be run in 01_install_requirements.sh?

Also I suspect there's a bunch of stuff we can remove from there, again to align with this new ansible based approach?

In the (likely) event there are additional packages needed for dev-scripts beyond those installed via metal3-dev-env, it would be nice if we could add an optional "extra_packages" interface to install-package-playbook.yml, so then we can just maintain a list of extra packages here and get rid of all the bash yum install stuff?

Reply:
Sure, I'll update this to move where this runs and remove some of the duplicate packages, at least as a first step